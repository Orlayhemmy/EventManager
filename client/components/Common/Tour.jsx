import React from 'react'; // eslint-disable-line

const tour = props => (
  <div id="tour" className="tour pagination-icon w-auto" onClick={props.tour}>
    <i className="fa fa-bus-alt" />
    <p class="tooltiptext">Take a tour</p>
  </div>
);
export default tour;
