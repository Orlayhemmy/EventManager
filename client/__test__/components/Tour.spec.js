import React from 'react';
import Tour from '../../components/Common/Tour';

describe('Tour Component', () => {
  it('should render component without error', () => {
    const wrapper = renderer
      .create(<Tour />)
      .toJSON();
    expect(wrapper).toMatchSnapshot();
  });
});
