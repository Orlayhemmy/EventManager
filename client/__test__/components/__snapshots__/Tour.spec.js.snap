// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tour Component should render component without error 1`] = `
<div
  className="tour pagination-icon w-auto"
  id="tour"
>
  <i
    className="fa fa-bus-alt"
  />
  <p
    className="tooltiptext"
  >
    Take a tour
  </p>
</div>
`;
